### PR TITLE
feat(stories): Migration to CSF format - FRONT-1117

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

**Comment by kalinchernev** [comment]
Please resolve conflicts

**Comment by planctus** [comment]
@kalinchernev, conflicts fixed ;)

File: babel.config.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
 module.exports = {
   presets: ['@babel/preset-env', '@babel/preset-react'],
-  plugins: ['preval'],
+  plugins: [
+    [
+      '@babel/transform-runtime',
```
Comment (kalinchernev) [inline babel.config.js:5]: why this change?

Comment (planctus) [reply babel.config.js:5]: That was needed for the script to migrate to run. But i have seen many times an error in the console for the missing regenerator-runtime in certain situations, so i thought that this would also be the way to fix that. Unfortunately i don't have "proof" about this, because i cannot reproduce the issue i described.
So if you think there are downsides of using this plugin we can remove it for now and reintegrate it only if we experience again the issue i mentioned.

Comment (kalinchernev) [reply babel.config.js:5]: For the script to run, ok. If it's not required after the work of the script and you're not sure about the benefits, I'd remove it, but it's up to you

Comment (planctus) [reply babel.config.js]: removed it @kalinchernev ;)
